data/json/snippets/survivor_notes.json

```diff
@@ -687,6 +687,169 @@
+      },
+      {
+        "id": "msn_note_20",
+        "//": "Thanks to Vetulus for helping me by providing this snippet idea.",
+        "text": "\"Why are their so many fucking giant spiders?\""
+      },
+      {
+        "id": "msn_note_21",
+        "//": "Thanks to Vetulus for helping me by providing this snippet idea.",
+        "text": "\"Partake of the fruit, eat the seed, accept the sap.\""
+      },
+      {
+        "id": "msn_note_22",
+        "//": "It's actually a thing to say 'You can't draw blood from a stone' - http://www.phrases.org.uk/meanings/you-cant-get-blood-out-of-a-stone.html",
+        "text": "\"I saw a rock moving, I took out my blood draw kit then drew blood from it.  Fuck you Vicesimus Knox!\""
+      },
+      {
+        "id": "msn_note_23",
+        "//": "Thanks to Lorith for helping me by providing this snippet idea.",
+        "text": "\"I knew I shoulda killed that spider in the basement.  Now there is a mob of giant spiders.  Fuck this place.\""
+      },
+      {
+        "id": "msn_note_24",
+        "//": "Thanks to Vetulus for helping me by providing this snippet idea.",
+        "text": "\"Do not go in the fucking basement my dude.\""
+      },
+      {
+        "id": "msn_note_25",
+        "//": "Thanks to Vetulus for helping me by providing this snippet idea.",
+        "text": "\"Call me at: 0406 650 430\""
+      },
+      {
+        "id": "msn_note_26",
+        "//": "Thanks to Lorith for helping me by providing this snippet idea.",
+        "text": "\"Fuck, I don't think this bug spray is going to work.\""
+      },
+      {
+        "id": "msn_note_27",
+        "//": "Thanks to 'TooFewSecrets' for helping me by providing this snippet idea.",
+        "text": "\"Those spores don't burn, but the rest of the mushrooms sure as Hell do.\""
+      },
+      {
+        "id": "msn_note_28",
+        "//": "Thanks to Aeron for helping me by providing this snippet idea.",
+        "text": "\"I saw some naked guy try to fistfight a hulk the other day.  He died, but someone who looked EXACTLY like him came by 5 minutes later attempting the same exact thing.  What the hell's going on there?\""
+      },
+      {
+        "id": "msn_note_29",
+        "//": "Thanks to Vetulus for helping me by providing this snippet idea.",
+        "text": "\"If by all laws of aviation a normal bee shouldn’t be able to fly, then how the fuck does a giant one?\""
+      },
+      {
+        "id": "msn_note_30",
+        "//": "Thanks to Lorith for helping me by providing this snippet idea.",
+        "text": "\"There was a turret on the corner yesterday.  I watched fifty people who looked the same run right up to it yelling \"SHOOT ME BITCH!\" before the last one punched it and died when it exploded.  Not five minutes later, another ran through the intersection like he knew it was safe.\""
```

Comment:
Too meta

Reply:
How so?

Reply:
It is a description of save scumming, which is 1. A thing we don't encourage.  2. It's clearly a description of a video game.  In fact it's a description of someone save scumming the game that you are playing right now when you find this note.  This could literally be an example of "metaknowledge"

Reply:
Would it work better if I reworded it to one of the drug notes? For example, someone saw this while high on acid, noting that they shouldn't take it again. Maybe it could be something portal storm related, such as someone seeing this during "weird weather"?